sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -463,6 +466,13 @@ The value horizontalCRS specifies the horizontal Coordinate Reference System. At
 The baseCRS is the geodetic CRS on which the projected CRS is based. In particular, the datum of the base CRS is also used for the derived CRS (see <<iho-s100,part=6,table=6>>).
 ====
 
+
+[[note12]]
+[NOTE]
+====
+This is the default vertical datum (verticalDatum), unless a distinct vertical datum(verticalDatum) is specified for the corresponding feature instance group (BathymetryCoverage.nn).
```

Comment:
`datum(verticalDatum)`
Add a space in between.

Reply:
done b7d5ead54bb2e04311fbd840167a0a3d19b89a21